src/plugins_3rdparty/primer3/src/Primer3Query.cpp

```diff
@@ -65,6 +60,12 @@ QString QDPrimerActor::getText() const {
 
 Task* QDPrimerActor::getAlgorithmTask(const QVector<U2Region>& /*location*/) {
     Task* t = nullptr;
+
+    auto settings = new Primer3TaskSettings;
```

Comment:
add () to the constructor call (use the same style everywhere in UGENE)

Reply:
✓